webapp/components/Editor/nodes/Hashtag.js

```diff
@@ -21,20 +21,30 @@ export default class Hashtag extends TipTapMention {
     return {
       ...super.schema,
       toDOM: node => {
+        // use a dummy domain because URL cannot handle relative urls
+        const url = new URL('/', 'https://human-connection.org')
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-09-17T06:43:20Z" title="Tuesday, September 17th 2019, 8:43:20 am +02:00">Sep 17, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -21,11 +21,15 @@ export default class Hashtag extends TipTapMention {
     return {
       ...super.schema,
       toDOM: node => {
+        // use a dummy domain because URL cannot handle relative urls
+        const url = new URL('/', 'http://example.org')
```


shouldn't we just use something else that can handle it then?

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-09-17T06:49:42Z" title="Tuesday, September 17th 2019, 8:49:42 am +02:00">Sep 17, 2019</time>_

---


actually, looking at the docs, and what you are actually doing here, if the only thing you are using is `url.search` then you could just use our domain instead of a dummy one

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-09-17T07:48:46Z" title="Tuesday, September 17th 2019, 9:48:46 am +02:00">Sep 17, 2019</time>_

---


Yes @mattwr18 , I would love to see our domain here as well! 😍

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-09-17T08:15:07Z" title="Tuesday, September 17th 2019, 10:15:07 am +02:00">Sep 17, 2019</time>_

---


What will be the network domain at the end?
`https://human-connection.org` is used for the donation and information page at the moment.
But I decided to change it to this. Hope it is okay for you @roschaefer !?